messaging/methods/media/listMedia.md

```diff
@@ -174,4 +181,30 @@ while True:
     if continuation_token is None:
         break
 ```
+
+{% sample lang="js" %}
+
+```js
+async function getMediaWithToken(continuationToken) {
+    await messagingController.listMedia(userId, continuationToken, function(error, response, context) {
+        console.log("Medias length: " + response.length);
+        console.log("Media 1 name: " + response[0].mediaName);
+        if (context.response.headers.hasOwnProperty('continuation-token')) {
+            continuationToken = context.response.headers['continuation-token'];
+        } else {
+            continuationToken = null;
+        }
+    });
+    return continuationToken;
```

Comment:
How do you get the `Media`?

Reply:
Or is just printing to the `console` what we are after?

Reply:
I'm fine for this example just printing to console but you do bring up a good point with using media elsewhere